src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +48,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	return &MobileClient{
+	mClient := &MobileClient{
 		Client:  client,
 		closed:  make(chan bool),
 		fronter: hqfd.NewDirectDomainFronter(),
 		appName: appName,
 	}
+	go mClient.updateConfig()
```

Comment:
@myleshorton We talked about moving the code to pull the configuration into its routine. I went ahead and did that and now I'm fetching it right away when we declare the client.